src/current/cockroachcloud/physical-cluster-replication.md

```diff
@@ -0,0 +1,256 @@
+- `physical_replication.sst_bytes`: The SST bytes (compressed) sent to the KV layer by all PCR streams.
+- `physical_replication.replicated_time_seconds`: The replicated time of the PCR stream in seconds since the Unix epoch.
+
+### Step 4. Fail over to the standby cluster
+
+Failing over from the primary cluster to the standby cluster will stop the PCR stream, reset the standby cluster to a point in time where all ingested data is consistent, and mark the standby as ready to accept application traffic. You can schedule the failover to:
+
+- The latest consistent time.
+- A time in the past within the [`retained_time`](#technical-reference).
+- A time up to 1 hour in the future.
+
+To specify a timestamp, send a `PATCH` request to the `/v1/replication-streams` endpoint along with the primary cluster, standby cluster, or the ID of the PCR stream. Include the `failover_at` timestamp and the `"status": "FAILING_OVER"` field:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --location --request PATCH "https://cockroachlabs.cloud/api/v1/replication-streams/7487d7a6-868b-4c6f-aa60-cc306cc525fe" --header "Authorization: Bearer api_secret_key" --header 'Content-Type:application/json' --data '{"status": "FAILING_OVER", "failover_at": "2025-01-13T19:35:14.472670Z"}'
+~~~
+
+To fail over to the latest consistent time, you only need to include `"status": "FAILING_OVER"` in your request with one of the cluster IDs or PCR stream ID:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --location --request PATCH "https://cockroachlabs.cloud/api/v1/replication-streams/7487d7a6-868b-4c6f-aa60-cc306cc525fe" --header "Authorization: Bearer api_secret_key" --header 'Content-Type:application/json' --data '{"status": "FAILING_OVER"}'
+~~~
+~~~json
+{
+  "id": "f1ea3b5a-5b9b-4101-9db4-780734554b14",
+  "status": "FAILING_OVER",
+  "source_cluster_id": "560cc798-7881-4a5b-baf0-321354a42c19",
+  "target_cluster_id": "24a9d47c-418c-4abc-940b-d2f665d95715",
+  "created_at": "2025-01-23T16:54:15.926325Z",
+  "retained_time": null,
+  "replicated_time": null,
+  "failover_at": null,
+  "activation_at": null
+}
+~~~
+
+After the failover is complete, both clusters can receive traffic and operate as separate clusters. It is necessary to redirect application traffic manually.
+
+{{site.data.alerts.callout_info}}
+PCR is on the cluster level, which means that the job also replicates all system tables. Users that need to access the standby cluster after failover should use the user roles for the primary cluster, because the standby cluster is a copy of the primary cluster. PCR overwrites all previous system tables on the standby cluster.
+{{site.data.alerts.end}}
+
+### Fail back to the primary cluster
+
+To fail back from the standby to the primary cluster, start another PCR stream with the standby cluster as the `sourceClusterId` and the original primary cluster as the `targetClusterId`. 
```

Comment:
Is it worth noting that we attempt to perform fast failback if possible, and fall back to regular failback if not?